src/lights.rs

```diff
@@ -0,0 +1,120 @@
+//! Contains different types of light sources.
+use gfx;
+
+use camera::Orthographic;
+use hub::Operation;
+use object::Object;
+use render::{BackendResources, ShadowFormat};
+
+/// `ShadowMap` is used to render shadows from [`PointLight`](struct.PointLight.html)
+/// and [`DirectionalLight`](struct.DirectionalLight.html).
+#[derive(Clone, Debug)]
+pub struct ShadowMap {
+    pub(crate) resource: gfx::handle::ShaderResourceView<BackendResources, f32>,
+    pub(crate) target: gfx::handle::DepthStencilView<BackendResources, ShadowFormat>,
+}
+
+#[derive(Clone, Debug)]
+pub(crate) enum ShadowProjection {
+    Ortho(Orthographic),
```

Comment:
These names ought to agree. I prefer the more verbose `Orthographic`.

Reply:
Makes sense, if we'll have `Perspective` in future.